resources/scripts/api/account/getTwoFactorTokenData.ts

```diff
@@ -0,0 +1,14 @@
+import http from "@/api/http";
+
+export interface TwoFactorTokenData {
+    image_url_data: string;
```

Comment:
I get a linting error here because it's not camel case, should I just make add `eslint-ignore` or?